bundles/core/src/main/java/com/adobe/cq/wcm/core/components/config/HtmlPageItemsConfig.java

```diff
@@ -0,0 +1,65 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.config;
```

Comment:
Shouldn't the `HtmlPageItemsConfig` interface be part of the `com.adobe.cq.wcm.core.components.internal` package, similar to the context aware configuration of the data layer?
The context aware config for the PDF Viewer component is also part of the internal package (`com.adobe.cq.wcm.core.components.internal.services.pdfviewer`).

Reply:
I actually think that was a bad choice and all configurations should be API, as we expect people to follow those structures. I was going to propose we find a way to migrate existing configuration to that package as well, in a backwards compatible way.

Reply:
As long as we treat all context aware configurations similarly I'm fine. Let's keep this one as it is and move the other ones to the same public package later.